mythril/laser/ethereum/transaction/transaction_models.py

```diff
@@ -191,12 +191,15 @@ def __init__(
         code=None,
         call_value=None,
         contract_name=None,
+        contract_address=None,
     ) -> None:
         self.prev_world_state = deepcopy(world_state)
         callee_account = world_state.create_account(
-            0, concrete_storage=True, creator=caller.value
+            0, concrete_storage=True, creator=caller.value, address=contract_address
```

Comment:
I think this might break if we supply world_state.create_account() with a hex address.

Reply:
Yes, `world_state.create_account` casts it to a BitVecVal, so it will need to be an integer